src/tutorial/json-editor/closing_thoughts.md

```diff
@@ -3,8 +3,8 @@
 This tutorial should get you started with a basic understanding of the flow of a `ratatui` program.
 However, this is only _one_ way to create a `ratatui` application. Because `ratatui` is relatively
 low level compared to other UI frameworks, almost any application model can be implemented. You can
-explore more of these in [Concepts: Application Patterns](./../../concepts/application-patterns.md)
-and get some inspiration for what model will work best for your application.
+explore more of these in [Concepts: Application Patterns](../../concepts/application-patterns) and
```

Comment:
Can we point it to the readme instead?

```suggestion
explore more of these in [Concepts: Application Patterns](../../concepts/application-patterns/README.md) and
```
```

Reply:
yep, this does work as a link btw in case that's not obvious.

Reply:
I didn't know that the worked but I also like the local files linked exactly so I can hit a keyboard shortcut and open the exact file :)

Reply:
makes sense - this is fixed